Client/mods/deathmatch/logic/lua/CLuaFunctionDefs.Util.cpp

```diff
@@ -24,6 +26,15 @@ int CLuaFunctionDefs::GetValidPedModels(lua_State* luaVM)
         }
     }
 
+    // gather our custom skin model IDs allocated with engineRequestModel
+    // (there might be some < 313 as well, and since we dont want duplicates, we start at 313, others are already included by the loop above)
```

Comment:
```suggestion
    // Gather our custom skin model IDs allocated with engineRequestModel
    // (there might be some < 313 as well, and since we don't want duplicates, we start at 313, others are already included by the loop above)
```